ubuntu-advantage.1

```diff
@@ -75,6 +75,48 @@ Possible values are: active, inactive, or n/a
 Show version of the Ubuntu Advantage package.
 
 
+.SH CONFIGURATION
+By default, ubuntu-advantage client configuration options are read from
+(/etc/ubuntu-advantage/uaclient.conf). If a (uaclient.conf) is present in
```

Comment:
Add i.e.: `(i.e. /etc/ubuntu-advantage/uaclient.conf). `

Why do we tell them about a local configuration file? I think we should omit that.

Reply:
will omit it. We probably shouldn't tell them thx.